lineman/app/components/flash/flash.haml

```diff
@@ -2,5 +2,3 @@
   .flash-container{ng-show: 'display()', ng-animate: "'enter'", class: 'alert-{{flash.level}}'}
     .flash-message
       {{ flash.message | translate:flash.options }}
-    %button.close.dismiss-flash.pull-right{type: 'button', ng-click: 'dismiss()'}
```

Comment:
Is there some code somewhere in here which makes the message actually go away? I guess I was expecting a javascript timeout or a css transition of some kind.